From 63c5f7a5960595db07493c06a57b7806f05def4f Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Sun, 17 Apr 2016 12:44:16 +0300
Subject: [PATCH 0348/2241] gic: a37xx: added GIC virtual maintenance interrupt

This interrupt is used by the KVM hypervisor to inject interrupts
to guest OS

Change-Id: I059cfdc1bab6daa47085a19183a98b9d1d46c95e
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29106
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 1a742b3..a58e7c8 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -301,6 +301,7 @@
 				compatible = "arm,gic-v3";
 				#interrupt-cells = <3>;
 				interrupt-controller;
+				interrupts = <GIC_PPI 9 (GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_HIGH)>;
 				reg = <0x1d00000 0x10000>, /* GICD */
 				      <0x1d40000 0x40000>; /* GICR */
 			};
-- 
2.7.4

